roles/datadog/tasks/pg_stats.yml

```diff
@@ -0,0 +1,54 @@
+---
+
+- name: enable postgres logging
+  set_fact:
+    datadog_checks:
+      postgres:
+        init_config:
+        instances:
+          - host: localhost
+            port: 5432
+            username: datadog
+            password: "{{ datadog_db_password }}"
+            dbname: "{{ db }}"
+            tags:
+              - "host:{{ inventory_hostname }}"
+            collect_activity_metrics: true
+        logs:
+          - type: file
+            path: /var/log/pg_log/postgresql-*.log
```

Comment:
Are we sure about that? As far as I read setting pg_log as the log destination stores them within the Postgres installation directory and not in /var/log.

Reply:
Check `log_directory` in https://github.com/openfoodfoundation/ofn-install/pull/410/files

Reply:
We havn't had any `stderr` output form UK prod yet, so we don't know... I'll check it again though

Reply:
https://docs.datadoghq.com/integrations/postgres/#log-collection

Reply:
You're right, so I've opted to go with the suggestions you quoted from here: https://logdna.com/postgresql-logging-guide/